LICENSE

```diff
@@ -0,0 +1,34 @@
+All Software contained in this document ("Software") is protected by copyright
+and is being made available under the "BSD License", included below. This
+Software may be subject to third party rights (rights from parties other than
+Ecma International), including patent rights, and no licenses under such third
+party rights are granted under this license even if the third party concerned
+is a member of Ecma International. SEE THE ECMA CODE OF CONDUCT IN PATENT
+MATTERS AVAILABLE AT http://www.ecma-international.org/memento/codeofconduct.htm
```

Comment:
Let’s use `https://` here?

Reply:
If you want that, I'd say you should PR the LIcense section of the spec first.

Reply:
Should we move the license section of the spec to this file, and have the build process dynamically include it, instead of having two sources of truth?

Reply:
https://github.com/bterlson/ecmarkup/pull/130

Reply:
@michaelficarra The upstream PR was merged. Can we update this PR too?